File: chap1/Taxes.java
// Ces programmes sont sous licence CeCILL-B V1.
class Taxes {
 
  public static void main (String [] args) {
    double ht,ttc;
    System.out.println("Quel est le prix hors taxes ?");
    ht = Isn.readDouble();
    ttc = ht + ht * 19.6 / 100;
    System.out.print("Le prix toutes taxes comprises est ");
    System.out.println(ttc);}}

File: chap1/Taxes.ml
(* Ces programmes sont sous licence CeCILL-B V1. *)
let _ =
  print_string "Quel est le prix hors taxes ?"; print_newline ();
  let ht = float_of_string (input_line stdin) in
  let ttc = ht +. ht *. 19.6 /. 100. in
  print_string "Le prix toutes taxes comprises est ";
  print_float ttc;
  print_newline ()


